config/webpack.config.js

```diff
@@ -124,7 +124,7 @@ module.exports = (env, argv) => {
         },
         {
           test: /\.(eot|svg|ttf|woff|woff2)$/,
-          use: ["file-loader"],
+          type: "asset",
```

Comment:
file-loader is deprecated in webpack 5. After upgrading css-loader, the font were not rendering correctly on buttons.